js/widgets/forms/autogrow.js

```diff
@@ -9,15 +9,16 @@ define( [
 	"jquery",
 	"./textinput" ], function( jQuery ) {
 //>>excludeEnd("jqmBuildExclude");
-(function( $, undefined ) {
+( function( $, undefined ) {
```

Comment:
lets not mix style updates into this we are planning on using esformater to do this all at once